src/routes/(auth)/inscription/+page.svelte

```diff
@@ -25,13 +26,15 @@
 			value={form?.password ?? ''}
 			name="password"
 			required
+			placeholder="**********"
```

Comment:
Not sure about this placeholder in terms of accessibility

Reply:
I've replace it. Better for the screen reader